### PR TITLE
Add Link and Profanity filtering

### PR DESCRIPTION
**Pull request by Mic-R**

### CODE REVIEW

**Comment by Mic-R** [comment]
@Nachwahl @Le4nderS 
Everything here is as we already told about, please test before merging!

File: src/events/message.js [hunk 1]
```diff
@@ -43,6 +49,77 @@ class MessageEvent extends BaseEvent {
 
                 cData[cData.step] = msg.content;
 
+                //moderate links and bad words
```
Comment (Qreepex) [inline src/events/message.js]: Muss an eine andere Stelle, ist aktuell mitten im Bewerbungskram drin, wird also nicht funktionieren

**Review by Qreepex — CHANGES_REQUESTED** [review]
Muss an eine andere Stelle im Message Event, ist aktuell mitten im Bewerbungskram drin.

**Comment by Mic-R** [comment]
Habe es jetzt mal überarbeitet, sollte so stimmen. Wenn ich es noch irgendwo reinhab bitte melden, denke aber das ist die Richtige Position.

**Review by robinferch — CHANGES_REQUESTED** [review]
Looks good so far, but you should consider switching to regex for the url checks to prevent users from get around the protection

**Comment by Mic-R** [comment]
> Looks good so far, but you should consider switching to regex for the url checks to prevent users from get around the protection

Ok, I will be doing that, but in the next change i will be pushing out with the tiktok/twitta/etc. api update

**Review by Qreepex — APPROVED** [review]
Looks good

**Comment by robinferch** [comment]
> > Looks good so far, but you should consider switching to regex for the url checks to prevent users from get around the protection
> 
> Ok, I will be doing that, but in the next change i will be pushing out with the tiktok/twitta/etc. api update

Please do that in a seperate PR

**Comment by Mic-R** [comment]
> requested

That was my plan anyways!

**Review by robinferch — APPROVED** [review]

**Pull request merged**